runtime/arc.js

```diff
@@ -104,7 +99,6 @@ export class Arc {
 
   dispose() {
     this._instantiatePlanCallbacks = [];
-    this._scheduler.unregisterArc(this);
```

Comment:
probably need to dispose the dataChangeCallbacks too?

Reply:
In general I think there are a lot of callbacks that remain wired even after dispose. This particular line I removed didn't seem to do anything in practice. I think I'll just drop in a TODO for now.